internal/rbd/rbd_util.go

```diff
@@ -79,6 +79,9 @@ const (
 
 	// krbd attribute file to check supported features.
 	krbdSupportedFeaturesFile = "/sys/bus/rbd/supported_features"
+
+	// CO cluster Name
+	COName = "CONAME"
```

Comment:
set this in format also instead of just CONAME it can be something like `csi.storage.k8s.io/co/name`